### DISCUSSION TITLE
Feature request: Add `__matmul__` to `sqlalchemy.sql.expression.ColumOperators`

### DISCUSSION THREAD

**Discussion by aradkdj** [sqlalchemy/sqlalchemy#11970] [Ideas]
Like `__{l,r}shift__`, `__matmul__` could be unused and provided for a custom operator to use `@` as an extension point. Currently GeoAlchemy uses `__{l,r}shift__` to render the respective `<<` and `>>` operators (see https://geoalchemy-2.readthedocs.io/en/stable/spatial_operators.html). `@` is another spatial operator with a corresponding Python operator, and not seeing it implemented vía `__matmul__` as well was surprising.

**Comment by CaselIT** [comment]
Hi,

A PR for this would be accepted.
